src/boot/vue-simple-acl.js

```diff
@@ -7,6 +7,7 @@ const userStore = useUserStore();
 const userRoles = computed(() => ({ roles: userStore.roles }));
 const rules = () => defineAclRules((setRule) => {
   setRule('create-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagram'));
+  setRule('create-template-diagram', (user) => !process.env.HAS_BACKEND || user.roles.includes('CF_createDiagramFromTemplate'));
```

Comment:
`create-template-diagram` -> `create-diagram-from-template`

Reply:
why ?

Reply:
I do what the issue says and if I remember you was present when we write it :)

Reply:
I agree with  @Zorin95670, no need to add the from. it will be too long and its already clear without the from

Reply:
The problem is it could be understand as "You can/can't create a new template diagram."...

Reply:
done